x/leverage/keeper/keeper_test.go

```diff
@@ -1255,16 +1257,6 @@ func (s *IntegrationTestSuite) TestBorrowAPYInvariant() {
 	// check invariant
 	_, broken = keeper.BorrowAPYInvariant(s.app.LeverageKeeper)(s.ctx)
 	s.Require().False(broken)
-
```

Comment:
Note: now that `SetBorrowAPY` errors on negative input, we would have to test for this condition some other way. The failure case isn't too critical though.